docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+version: "1"
+services:
+  app:
+    build:
+      context: .
+
+    env_file:
+      - .env
+
+    ports:
+      - "8000:8000"
+
+    volumes:
+      - ./:/app
+      - my_media:/files/media
+
+    command: >
+      sh -c "/app/wait_for_db.sh &&
+            python manage.py migrate &&
+            python manage.py runserver 0.0.0.0:8000"
+
+    depends_on:
+      - db
+
+
+  db:
+    image: postgres:16-alpine3.17
+
+    restart: always
+
+    env_file:
+      - .env
+
+    ports:
+      - "5432:5432"
+
+    volumes:
+      - my_db:$PGDATA
```

Comment:
The volume path `$PGDATA` is used here, which is an environment variable. Ensure that this variable is defined in your environment or consider using a static path for clarity.